scripts/deployment/deploy.sh

```diff
@@ -361,7 +361,9 @@ deploy_openlibrary() {
 # - deploy.sh olsystem
 # - deploy.sh openlibrary
 
-if [ "$1" == "olsystem" ]; then
+if [ "$1" == "crons" ]; then
+    check_crons
+elif [ "$1" == "olsystem" ]; then
```

Comment:
I wonder if we can roll this back now? If not, we should remove the check_cron calls from this script.